app/models.py

```diff
@@ -171,6 +171,28 @@ class Person(SearchableMixin, db.Model):
     education = db.Column(db.String)
     publications = db.Column(db.String)
 
+    def redact_data(self, fields):
+        if not fields["image"]:
+            self.image = None
+        if not fields["email"]:
+            self.email = None
+        if not fields["room"]:
```

Comment:
Given that we already censor room, I think we should exclude this

Reply:
AFAIK, this data is still included in API calls. Removing this wouldn't give folks a way to scrub this data from API data.

Reply:
I mean, the fields are there, but there isn't any data filling them. I don't see why we would support an input for that